### ISSUE TITLE
Feature Request: Respect more than one docker-compose file

### ISSUE THREAD

**Issue by maltemuth** [jplhomer/lifeboat#46]
In my setup, I do have

- a docker-compose.yml describing the services needed to run the app
- a docker-compose.dev.yml describing the services needed to develop the app (like build services and reloaders)
- a docker-compose.behat.yaml describing the services needed to test the app (in this case behat-specific

Lifeboat only shows the the services from docker-compose.yml; is it possible to recognize the other files as well?

**Comment by jplhomer** [comment]
Yes! This seems like a good feature idea. A feature to select which compose files to run would be beneficial. We'll probably need to support a cascading specificity for files too - since I assume they override each other based on the order you apply them.
